test/common/network/address_impl_test.cc

```
@@ -288,6 +289,20 @@ TEST(PipeInstanceTest, Basic) {
   EXPECT_EQ(nullptr, address.ip());
 }
 
+TEST(PipeInstanceTest, AbstractNamespace) {
+  std::unique_ptr<PipeInstance> address;
+#if defined(__linux__)
+  address = std::make_unique<PipeInstance>("@/foo");
+#else
+  EXPECT_THROW(address = std::make_unique<PipeInstance>("@/foo"), EnvoyException);
+  return;
+#endif
+  ASSERT_NE(address, nullptr);
```

No need to assert this; just let it crash on the next line if it is NULL.